src/dpr/routes/asyncReports.ts

```diff
@@ -244,7 +244,7 @@ export default function routes({
   // 3 - VIEw REPORT
   const viewReportPaths = [
     '/async/:type/:reportId/:id/request/:tableId/report',
-    '/async/:type/:reportId/:id/request/:tableId/report/download',
+    '/async/:type/:reportId/:id/request/:tableId/report/:download',
```

Comment:
@MOJ-Fordie The "/download" URL search in App Insights will need to change to "/downloadReport/" once this change goes live.